webapp/channels/src/components/channel_selector_modal/channel_selector_modal.tsx

```diff
@@ -236,9 +235,12 @@ export class ChannelSelectorModal extends React.PureComponent<Props, State> {
                         componentClass='h1'
                         id='channelSelectorModalLabel'
                     >
-                        <FormattedMarkdownMessage
-                            id='add_channels_to_scheme.title'
-                            defaultMessage='Add Channels to **Channel Selection** List'
+                        <FormattedMessage
+                            id='manage_scheme_channels.title'
```

Comment:
Nit: let's change the id to "channelSelectorModal.title", you have to delete the translation in en file first and run the extract command again

Reply:
Done!